lib/monitor.ts

```diff
@@ -112,9 +112,8 @@ function getAddress(req: any) : string {
  * @param {[number,number]} start the start time of the dependecy request
```

Comment:
@KarineValenca, please improve the documentation to explain the consequences of `errorMessage` being empty vs filled